assets/web/pin.html

```diff
@@ -28,14 +28,14 @@ <h1 class="my-4">PIN Pairing</h1>
     fetch("/api/pin", { method: "POST", body: b })
       .then((response) => response.json())
       .then((response) => {
-        if (response.status) {
+        if (response.status && response.status === "true") {
```

Comment:
Probably only need one or the other here.

I don't know exactly `response.status` is returning. If it's a string then the second one is probably what is needed over the first one as `if (response.status)` would be Truthy if any string is present.

https://stackoverflow.com/a/23061950/11214013

Also, if `response.status` is a string, it might be worth changing to `response.status.toLowerCase()`

Assuming a string, I'd suggest:
```javascript
if (response.status.toLowerCase() === "true") {
```

Reply:
I decided to make it more explicit by combining both toString and toLowerCase